cmd/soroban-rpc/internal/test/simulate_transaction_test.go

```diff
@@ -22,7 +25,17 @@ var (
 	testSalt     = sha256.Sum256([]byte("a1"))
 )
 
-// createInvokeHostOperation creates a dummy InvokeHostFunctionOp. In this case by installing a contract code.
+func getHelloWorldContract(t *testing.T) []byte {
+	_, filename, _, _ := runtime.Caller(0)
+	testDirName := path.Dir(filename)
+	contractFile := path.Join(testDirName, "../../../../target/wasm32-unknown-unknown/test-wasms/test_hello_world.wasm")
+	ret, err := os.ReadFile(contractFile)
+	if err != nil {
+		t.Fatalf("test_hello_world.wasm can't be found (%v) please run `make build-test-wasms` at the project root directory", err)
```

Comment:
not always. Yes, there was an error reading the file, and "file not found" is very likely the reason.
could you change the text to "unable to read test_hello_world.wasm due to %v" ? Suggesting to run `build-test-wasms` is probably a good solution in the case that the file not found.